docs/architecture/adr/ADR030-aws-service-operator.md

```diff
@@ -0,0 +1,29 @@
+# ADR030: AWS Service Operator
+
+## Status
+
+Pending
+
+## Context
+
+[Amazon announced](https://aws.amazon.com/blogs/opensource/aws-service-operator-kubernetes-available/) an AWS Service Operator for Kubernetes in October 2018.
+There is a need for one or more users to provision queues (potentially SQS ones) for their application to interact with.
+The AWS Service Operator consists of a container that sits in the Kubernetes cluster and monitors for custom resource types which it maps to the appropriate CloudFormation resources and deploys.
+It supports the following resources:
```

Comment:
Should this be future-proofed? 

Meaning, is there a link where an exhaustive list exists? 
It might be worth explicitly mentioning that these are the resources that are supported _right now_, and have a link to an AWS source to outline what may be supported at a later time when this might be read.

Reply:
Well we effectively have a fork of aws-service-operator's config which defines the CRDs and therefore which resources people in our cluster can use, and also specifies the exact version of the aws-service-operator image.
So if that changed upstream we'd still have to make gsp.git changes, and could update this list.

Reply:
https://github.com/awslabs/aws-service-operator/issues/190